site/docs-md/cordova/migrating-from-cordova-to-capacitor.md

```diff
@@ -0,0 +1,33 @@
+# Migrating from Cordova/PhoneGap to Capacitor
+
+Capacitor can be used as a replacement for Cordova/PhoneGap in many cases. The two projects provide a web environment with full access to native features and functionality, but with a few differences in their approach to tooling, native project management, and plugins. Capacitor also has support for existing Cordova plugins, providing a new runtime with access to the rich Cordova plugin ecosystem.
+
+## Using Cordova Plugins in Capacitor
+
+Capacitor was designed from the start to support the rich Cordova plugin ecosystem out of the box. Thus, using Cordova plugins in Capacitor is easy. Follow the [Cordova Plugin]() guide for more information on how this works.
+
+# Using Cordova Plugins
+
+Capacitor comes with Cordova plugin compatibility, so using existing Cordova plugins in Capacitor is easy.
+
+## Installing Cordova Plugins
+
+Simply `npm install` your plugin of choice, sync your project, finish configuration, and you're ready to go:
+
+```bash
+npm install cool-cordova-plugin
+npx cap sync
+```
+
+## Important: Configuration 
+
+Capacitor does not support Cordova install variables, auto configuration, or hooks, due to our philosophy of letting you control your native project source code (meaning things like hooks are unnecessary). If your plugin requires variables or settings to be set, you'll need to apply those configuration settings manually by mapping between the plugin's `plugin.xml` and required settings on iOS and Android.
+
+Consult the [iOS](../ios/configuration) and [Android](../android/configuration) configuration guides for info on how to configure each platform.
+
+## Compatibility Issues
```

Comment:
Recommend linking somewhere in this section back to the Incompatible Plugins page.